src/frame/tests/index.test.ts

```diff
@@ -632,15 +632,15 @@ describe("frame index", () => {
 
             const payload: MT.GroupCreateRequest = {
                 type: "GROUP_CREATE",
-                message: {groupID: "my-group", groupName: "bar", addAsMember: true, needsRotation: false},
+                message: {groupID: "my-group", groupName: "bar", addAsMember: true, needsRotation: false, memberList: ["string"]},
```

Comment:
Very minor, but I'd probably change "string" to something like "userID" so that it's a lot more clear about what type of data you're passing.

Reply:
done